### PR TITLE
chore(doc): Added upgrade doc and adopters list for lvm-localpv

### PR DESCRIPTION
**Pull request by w3aman**
Signed-off-by: w3aman <aman.gupta@mayadata.io>

- This PR adds adopters list and upgrade steps for lvm-localpv

### CODE REVIEW

File: upgrade/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+*Prerequisite*
```
Comment (pawanpraka1) [inline upgrade/README.md:1]: make it a header. add ### in the beginning

File: Adopters.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# OpenEBS LVM-LocalPV Adopters
+
+This is the list of organizations and users that publicly shared details of how they are using OpenEBS LVM-LocalPV CSI driver for running their Stateful workloads. Please send PRs to add or remove organizations/users.
+
+[click here](https://github.com/openebs/openebs/issues/2719) to see the list of organizations/users who have publicly shared the usage of OpenEBS.
+
+The list of organizations that have publicly shared the usage of LVM-LocalPV.
```
Comment (pawanpraka1) [inline Adopters.md:7]: let us remove this for now as there is no adopters. Will add this section later.

File: Adopters.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# OpenEBS LVM-LocalPV Adopters
+
+This is the list of organizations and users that publicly shared details of how they are using OpenEBS LVM-LocalPV CSI driver for running their Stateful workloads. Please send PRs to add or remove organizations/users.
```
Comment (pawanpraka1) [inline Adopters.md:3]: do we need this line?

Comment (w3aman) [reply Adopters.md]: removed

**Review by pawanpraka1 — APPROVED** [review]
looks good.

**Pull request merged**